mod_reforged/hooks/experimental_modules/onDropLoot.nut

```diff
@@ -0,0 +1,60 @@
+// Original feature request: https://github.com/MSUTeam/MSU/issues/220
+
+// Goal:
+// - Add a new function to all actors which can be used to modify the existing loot they drop on death, or to add new loot.
+
+::Reforged.HooksMod.hook("scripts/entity/tactical/actor", function(q) {
+	q.onDropLoot <- function( _originalLoot, _killer, _skill, _tile, _fatalityType )
```

Comment:
A problem is when we have let's say 2 mods who both want to drop everything from the `_originalLoot` except `item.my_item`. So they both hook this function and iterate over the `_originalLoot` and drop everything manually except that item type. This will cause that item to be dropped twice.

Instead of this perhaps it'd be better to have a function like `getLoot` which is passed the `_originalLoot` and then mods can hook/modify that function to add/remove things to the loot. Then finally the loot is dropped by an internal function. Using this system the above example is solved because both mods modify the `_loot` array and remove `item.some_item` from it. They don't care about manually dropping the rest and leave that up to the framework.